internal/slices/slices.go

```diff
@@ -0,0 +1,29 @@
+package slices
+
+import (
+	"golang.org/x/exp/constraints"
+)
+
+// Min returns the minimum value in a list.
+// It will panic if no arguments are provided.
+func Min[T constraints.Ordered](list ...T) T {
+	min := list[0]
+	for _, val := range list {
```

Comment:
Since we already visited the first element in line 10, I think we can skip it in the for loop

```suggestion
	for _, val := range list[1:] {
```

Reply:
Then your list requires at least two elements. I think saying `Min([1])` should be perfectly fine.

Reply:
Okay I found a nice middleground for this by just iterating using `for i := 1; i < len(list); i++`

Reply:
> Then your list requires at least two elements. I think saying Min([1]) should be perfectly fine.
No it does not. With that, if the list only has 1 element, it does not enter the loop, since list[1:] is empty.